src/engraving/style/style.cpp

```diff
@@ -392,6 +418,86 @@ void MStyle::read(XmlReader& e, compat::ReadChordListHook* readChordListHook)
+            set(Sid::systemTextPosBelow, PointF(e.readPoint()));
+        } else if (tag == "systemMinDistance") { // pre-x.y typo
+            set(Sid::systemTextMinDistance, Spatium(e.readDouble()));
+        } else if (tag == "systemFrameType") { // pre-x.y typo
+            set(Sid::systemTextFrameType, e.readInt());
+        } else if (tag == "systemFramePadding") { // pre-x.y typo
+            set(Sid::systemTextFramePadding, e.readDouble());
+        } else if (tag == "systemFrameWidth") { // pre-x.y typo
+            set(Sid::systemTextFrameWidth, e.readDouble());
+        } else if (tag == "systemFrameRound") { // pre-x.y typo
+            set(Sid::systemTextFrameRound, e.readInt());
+#if 0
+        } else if (tag == "systemFrameFgColor") { // pre-x.y typo
+            set(Sid::systemTextFrameWidth, );
+        } else if (tag == "systemFrameBgColor") { // pre-x.y typo
+            set(Sid::systemTextFrameBgColor, );
+#endif
+        } else if (tag == "staffFontFace") { // pre-x.y typo
+            String staffFontFace = e.readText();
+            if (!staffFontFace.empty()) {
+                set(Sid::systemTextFontFace, staffFontFace);
+            }
+        } else if (tag == "staffFontSize") { // pre-x.y typo
+            set(Sid::staffTextFontFace, e.readDouble());
+        } else if (tag == "staffFontSpatiumDependent") { // pre-x.y typo
+            set(Sid::staffTextFontSpatiumDependent, bool(e.readInt()));
+        } else if (tag == "staffFontStyle") { // pre-x.y typo
+            set(Sid::staffTextFontStyle, e.readInt());
+#if 0
+        } else if (tag == "staffAlign") { // pre-x.y typo
+            set(Sid::staffTextAlign, );
+#endif
+        } else if (tag == "staffOffsetType") { // pre-x.y typo
+            set(Sid::staffTextOffsetType, e.readInt());
+        } else if (tag == "staffPosAbove") { // pre-x.y typo
+            set(Sid::staffTextPosAbove, PointF(e.readPoint()));
+        } else if (tag == "staffPosBelow") { // pre-x.y typo
+            set(Sid::staffTextPosBelow, PointF(e.readPoint()));
+        } else if (tag == "systemMinDistance") { // pre-x.y typo
+            set(Sid::systemTextMinDistance, Spatium(e.readDouble()));
+        } else if (tag == "staffFrameType") { // pre-x.y typo
+            set(Sid::staffTextFrameType, e.readInt());
+        } else if (tag == "staffFramePadding") { // pre-x.y typo
+            set(Sid::staffTextFramePadding, e.readDouble());
+        } else if (tag == "staffFrameWidth") { // pre-x.y typo
+            set(Sid::staffTextFrameWidth, e.readDouble());
+        } else if (tag == "staffFrameRound") { // pre-x.y typo
+            set(Sid::staffTextFrameRound, e.readInt());
+#if 0
+        } else if (tag == "staffFrameFgColor") { // pre-x.y typo
+            set(Sid::staffTextFrameFgColor, );
+        } else if (tag == "staffFrameBgColor") { // pre-x.y typo
+            set(Sid::staffTextFrameBgColor, );
+#endif
+        } else if (tag == "dymanicsShowTabCommon") { // pre-x.y typo
```

Comment:
Seems to be an issue with GuitarPro tests (they report, not sure whether these all need mending or whether this was a typo in the code once upon a time.